### PR TITLE
fix: changing card hover behaviour

### PR DESCRIPTION
**Pull request by alok-traceable**
## Description
Global changes on card component hover behavior.
- Removing grey background on hover
- Changes on box-shadow on hover
- Removing change in border color on hover

### Testing
Verified on the local environment. Mostly all changes are on stylings.

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/assets-library/assets/styles/_interaction.scss [hunk 1]
```diff
@@ -9,12 +9,16 @@
 
 @mixin card-hover {
   &:hover {
-    @include dropdown-box-shadow();
+    @include box-shadow-2dp();
     cursor: pointer;
   }
 }
 
 @mixin filter-box-shadow {
```
Comment (aaron-steinfeld) [inline projects/assets-library/assets/styles/_interaction.scss:17]: nit: why the 2 separate mixins defined the same?

**Review by aaron-steinfeld — APPROVED** [review]

**Comment by anandtiwary** [comment]
@alok-traceable Could you please share a before and after screenshot?

File: projects/observability/src/shared/components/card-list/card-list.component.scss [hunk 2]
```diff
@@ -33,21 +34,13 @@
       }
 
       &:hover {
-        background: $blue-2;
```
Comment (anandtiwary) [inline projects/observability/src/shared/components/card-list/card-list.component.scss:36]: We don't want to show a darker blue when user hover on selected card?

Comment (alok-traceable) [reply projects/observability/src/shared/components/card-list/card-list.component.scss:36]: Need to remove these on hover on card
1. border
2. background color change

To maintain consistency on hover elements, selected and un-selected cards should follow the same behavior.
I can revert this change if required.

**Pull request merged**